### ISSUE TITLE
The "Set Remote Variable" graph node has a "Graph Remote Variable Ref" connector on the "Entity" input

### ISSUE THREAD

**Issue by Tisten** [OurMachinery/themachinery-public#1420]
- **Version:** Git Master Branch
- **OS:** Windows

I guess this is correct. But it becomes confusing when all other connectors called "Entity" are of the "Runtime Entity" type.
<img width="327" alt="image" src="https://user-images.githubusercontent.com/6726075/175090011-d7bd450f-3bcb-4918-b650-6f7b1ba9aea3.png">

My spontaneous reaction when I see this is that it should be possible to connect the entity where I want to set the remote variable. Since I've been part of the discussion around how the variables are stored I understand that the connector isn't usable today, but it still cause confusion.

I'm thinking that since there currently isn't any outputs which generate a "Graph Variable Ref" or "Graph Remote Variable Ref" wire, perhaps there doesn't need to be any connector? Or the UI for that type could be changed to a button which creates a modal dialouge where the variable can be found?

**Comment by simon-ourmachinery** [comment]
I agree this is a unlucky UX / UI problem I will look into how this can be solved. This is not just a issue for this node for any TT type that's used in the graph that expands like this.
